README.md

```diff
@@ -22,20 +22,16 @@ Your Rails application needs to use Webpacker and have the React integration don
 First, you need to add the webpacker-react gem to your Rails app Gemfile:
 
 ```ruby
-gem 'webpacker-react', "~>0.1.0"
+gem 'webpacker-react', "~> 0.1.0"
 ```
 
-Once done, run `bundler` to install the gem.
+Once done, run `bundle` to install the gem.
 
-Then you need to update your `vendor/package.json` file to include the `webpacker-react` NPM module:
-```json
-  "dependencies": {
-    "..."
-    "webpacker-react": "~>0.1.0"
-  },
-```
+Then you need to update your `package.json` file to include the `webpacker-react` NPM module:
+
+`./bin/yarn add webpacker-react@0.1.x`
```

Comment:
`./bin/yarn add webpacker-react` adds `"webpacker-react": "^0.1.0",` to the `package.json`, which is better here.